### PR TITLE
DOCSP-41078 - v6.8

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-41078
Staging - https://preview-mongodbmongokart.gatsbyjs.io/node/docsp-41078-v6.8/whats-new/#std-label-version-6.8

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -58,6 +59,69 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.8:
+
+What's New in 6.8
+-----------------
+
+The {+driver-short+} v6.7 release includes the following features:
```
Comment (jordan-smith721) [inline source/whats-new.txt:67]:
```suggestion
The {+driver-short+} v6.8 release includes the following features:
```

File: source/whats-new.txt [hunk 2]
```diff
@@ -58,6 +59,69 @@ Learn what's new in:
...
+
+- Fixes a bug where a local KMS provider accepted a ``BSON Binary`` instance at
+  runtime, but the TypeScript compiler allowed only values of type ``Buffer`` and
+  ``string``.
+
+- The ``ReadConcernMajorityNotAvailableYet`` error is now a retryable read error.
+
+- KMS providers can now be associated with a name, and multiple keys can be provided
+  for each KMS provider. This feature requires ``mongodb-client-encryption`` v6.0.1 or later.
```
Comment (jordan-smith721) [inline source/whats-new.txt:76]:
```suggestion
- You can now associate a name and provide multiple keys for KMS providers. This feature requires ``mongodb-client-encryption`` v6.0.1 or later.
```

File: source/whats-new.txt [hunk 3]
```diff
@@ -58,6 +59,69 @@ Learn what's new in:
...
+     'aws:key1': {
+       accessKeyId: ...,
+       secretAccessKey: ...
+     },
+     'aws:key2': {
+       accessKeyId: ...,
+       secretAccessKey: ...
+     },
+  
+   clientEncryption.createDataKey('aws:key-1', { ... });
```
Comment (jordan-smith721) [inline source/whats-new.txt:96]: Indent this so it's in the last bullet's section

File: source/whats-new.txt [hunk 4]
```diff
@@ -58,6 +59,69 @@ Learn what's new in:
...
+  option is set to ``true``, the KMIP provider performs encryption and decryption of
+  the data key locally, ensuring that the encryption key never leaves the KMIP server.
+  This feature requires ``mongodb-client-encryption`` v6.0.1 or later.
+  
+  The following code example shows how to specify this option:
+
+.. code-block:: javascript
+   :copyable: true
+
+   clientEncryption.createDataKey('kmip', { masterKey: { delegated: true } } );
```
Comment (jordan-smith721) [inline source/whats-new.txt:108]: Indent this one too

File: source/whats-new.txt [hunk 5]
```diff
@@ -58,6 +59,69 @@ Learn what's new in:
...
+
+- The driver now decodes BSON responses as the cursor iterates over them,
+  rather than decoding the entire BSON response when it is received.
+
+- The Github release for the ``mongodb`` package now contains a detached signature file for
+  the NPM package (named ``mongodb-X.Y.Z.tgz.sig``). This change applies to every major
```
Comment (jordan-smith721) [inline source/whats-new.txt:114]: I don't think these parenthesis are necessary
```suggestion
  the NPM package ``mongodb-X.Y.Z.tgz.sig``. This change applies to every major
```

File: source/whats-new.txt [hunk 6]
```diff
@@ -58,6 +59,69 @@ Learn what's new in:
...
+  and patch release for versions 5 and 6 of the driver. To verify the package signature,
```
Comment (jordan-smith721) [inline source/whats-new.txt:115]: S: 
```suggestion
  and patch release for versions 5.x and 6.x of the driver. To verify the package signature,
```

**Review by jordan-smith721 — CHANGES_REQUESTED** [review]
Just a couple suggestions

File: source/whats-new.txt (hunk 1 above)
Comment (mongoKart) [reply source/whats-new.txt:67]: whew. good catch

File: source/whats-new.txt (hunk 5 above)
Comment (mongoKart) [reply source/whats-new.txt:114]: i think the file name refers to the signature file, not the NPM package. reworded anyway

Comment (mongoKart) [inline source/whats-new.txt:114]:
```suggestion
- The Github release for the ``mongodb`` package now contains a detached signature file,
   ``mongodb-X.Y.Z.tgz.sig``, for the NPM package. This change applies to every major
```

File: source/whats-new.txt [hunk 7]
```diff
@@ -58,6 +59,68 @@ Learn what's new in:
...
+
+- The driver now decodes BSON responses as the cursor iterates over them,
+  rather than decoding the entire BSON response when it is received.
+
+- The Github release for the ``mongodb`` package now contains a detached signature file,
+   ``mongodb-X.Y.Z.tgz.sig``, for the NPM package. This change applies to every major
+  and patch release for versions 5.x and 6.x of the driver. To verify the package signature,
+  follow the instructions in the Release Integrity section of the
+  `README.md <https://github.com/mongodb/node-mongodb-native/blob/v6.8.0/README.md#release-integrity>`__
+  file in the driver's GitHub repository.
```
Comment (mongoKart) [inline source/whats-new.txt:117]:
```suggestion
- The Github release for the ``mongodb`` package now contains a detached signature file,
  ``mongodb-X.Y.Z.tgz.sig``, for the NPM package. This change applies to every major
  and patch release for versions 5.x and 6.x of the driver. To verify the package signature,
  follow the instructions in the Release Integrity section of the
  `README.md <https://github.com/mongodb/node-mongodb-native/blob/v6.8.0/README.md#release-integrity>`__
  file in the driver's GitHub repository.
```

File: source/whats-new.txt [hunk 8]
```diff
@@ -58,6 +59,69 @@ Learn what's new in:
...
+
+The {+driver-short+} v6.8 release includes the following features:
+
+- Fixes a bug where a local KMS provider accepted a ``BSON Binary`` instance at
+  runtime, but the TypeScript compiler allowed only values of type ``Buffer`` and
+  ``string``.
+
+- The ``ReadConcernMajorityNotAvailableYet`` error is now a retryable read error.
+
+- You can now associate a name, and provide multiple keys for, KMS providers.
```
Comment (jordan-smith721) [inline source/whats-new.txt:75]:
```suggestion
- You can now associate a name, and provide multiple keys for KMS providers.
```

**Review by jordan-smith721 — APPROVED** [review]
LGTM with one thing. It looks like the API documentation hasn't been updated to 6.8 yet though so we might want to wait to deploy this

**Pull request merged**